_plugins/sidebar.rb

```diff
@@ -4,6 +4,11 @@
 require 'nokogiri'
 require 'json'
 
+###
+# This plugin works only if used as a remote-theme
+# If included as a theme or gem this plugin must be copied over
```

Comment:
Hm, this is unfortunate. Will perhaps moving the `sidebar.rb` file somewhere else help?

Reply:
I am unsure as to how this will help.
The file is included on disk, it's not loaded by Jekyll however.

Reply:
I was thinking that if we made the code available in the Gem as recommended for Ruby Gems, chances may be that the code will be loaded. I don't think a theme differs from other Ruby Gems in this respect. Just try to do the same we've done in the other Gems we've created.